dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.xmlwsSecurity-3.0

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.xmlwsSecurity-3.0
+visibility=private
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.xmlWS-3.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurity-4.0)))"
+-bundles=com.ibm.ws.jaxws.2.3.security.jakarta
```

Comment:
This should be com.ibm.ws.jaxws.2.3.security.

Reply:
Nice catch, that was a bad merge on my part after the code reviews to the other PR.